src/tomography.jl

```diff
@@ -0,0 +1,244 @@
+using Shortcuts, SCS, QIP, Convex
+
+function tomo_gate_set(nbrQubits, nbrPulses; pulse_type="Clifford", prep_meas = 1)
+#tomo_gate_set Returns a set of state preparation or readout unitary gates.
+
+#pulse_type: prepared states/meas. axes
+#prep_meas: 1 for prep., 2 for meas. pulses
+
+#Basic Pauli operators
+    X = [[0 1]; [1 0]];
+    Y = [[0 -1im]; [1im 0]];
+    Z = [[1 0]; [0 -1]];
+I = eye(2);
+Uset1Q = Dict()
+gateSet = Dict()
```

Comment:
Unfortunately, this construction gives Julia no information about the types of elements of `Uset1Q` and `gateSet`. If you really want them to be `Dict`s, then make these declarations `Dict{Int,Matrix{Complex128}}()`.